packages/mdc-theme/README.md

```diff
@@ -236,3 +236,16 @@ It only returns the raw color value of the specified theme property.
 @debug mdc-theme-prop-value(primary); // #3f51b5
 @debug mdc-theme-prop-value(blue);    // blue
 ```
+
+#### `mdc-theme-accessible-ink-color($fill-color)`
```

Comment:
mdc-theme-accessible-primary-ink-color? Since it always returns primary text colors?

Reply:
Good call! I made it a parameter called `$text-style`, which defaults to `primary`.